Dream-Home-PEABOT/FEE_DreamHome
# Om feature display report

### What does this PR do?
- it creates a Report component
- it styles <Report>
- it adds s category section to the report component
### Where should the reviewer start?
- `src/components/Report/Report.css`
- `src/components/Report/Report.tsx`

### How should this be manually tested?
- run `npm` start, the application  should start on `localhost/3000` and the Report component should render at the bottom of the application
### Any background context you want to provide?

### What are the relevant tickets?
closes #43 
### Questions:

### Screenshots (if appropriate)
<img width="1433" alt="Screen Shot 2021-01-07 at 5 31 59 PM" src="https://user-images.githubusercontent.com/56229864/103959929-41c8c200-510e-11eb-9687-60f3c96319a7.png">
<img width="1433" alt="Screen Shot 2021-01-07 at 5 32 09 PM" src="https://user-images.githubusercontent.com/56229864/103959934-45f4df80-510e-11eb-8ddf-8d1e52e9a5f4.png">
